src/util-inl.h

```diff
@@ -445,6 +445,33 @@ v8::MaybeLocal<v8::Value> ToV8Value(v8::Local<v8::Context> context,
   return v8::Number::New(isolate, static_cast<double>(number));
 }
 
+template <typename T>
+v8::Local<v8::Array> ToV8ValuePrimitiveArray(v8::Local<v8::Context> context,
+                                             const std::vector<T>& vec,
+                                             v8::Isolate* isolate) {
+  if (isolate == nullptr) isolate = context->GetIsolate();
+  v8::EscapableHandleScope handle_scope(isolate);
+  v8::Local<v8::Array> arr = v8::Array::New(isolate, vec.size());
```

Comment:
Using a `v8::LocalVector` and creating the `v8::Array` and the end without using the `Set(...)` method would be better here.